### PR TITLE
Fix EZP-24790: Format server-side PJAX errors using PlatformUI base template

### PR DESCRIPTION
**Pull request by lolautruche**
> https://jira.ez.no/browse/EZP-24790

Implemented new `ExceptionListener` / `ExceptionController` based on the Symfony ones.
They will only trigger if exceptions are coming from a PJAX request.

Exception style has been kept to enable preview from one's browser debug console (with stacktrace).
Please note that full exception description, including stack trace, is only displayed with debug on (usually in dev environment).

Depends on #323 and https://github.com/ezsystems/ezpublish-kernel/pull/1405

### CODE REVIEW

**Comment by andrerom** [comment]
+1

**Comment by yannickroger** [comment]
If I understand correctly, error messages are the ones returned by the kernel and they are not translated anymore.

**Comment by lolautruche** [comment]
Yes indeed, but that should not be a problem. A not found is a not found

File: Resources/views/Section/not_found.html.twig [hunk 1]
```diff
@@ -1,14 +0,0 @@
-{% extends "eZPlatformUIBundle::pjax.html.twig" %}
-
-{% trans_default_domain "section" %}
-
-{% block title %}{{ 'section.not_found.title'|trans }}{% endblock %}
```
Comment (yannickroger) [inline Resources/views/Section/not_found.html.twig]: these translation keys (title and message) could be removed from: `Resources/translations/section.en.xlf`

**Comment by yannickroger** [comment]
Code looks good. I am just a bit concerned by the loss of internationalization/customization. ping @dpobel @andrerom 
Once that sorted out: +1

**Comment by lolautruche** [comment]
@yannickroger About i18n of exceptions, this is a kernel problem. Repository exceptions are currently not translatable (may you report that on Jira please?)

**Comment by dpobel** [comment]
@yannickroger that's a very valid concern.
Now the question is, how should we handle this problem ?

**Comment by lolautruche** [comment]
@dpobel @yannickroger AFAICT, this can only be handled inside the repository, by making these exceptions translatable...

**Comment by lolautruche** [comment]
OK, PR is now complete. All not found / access checks are left to repository + symfony now.

Still missing a +1 :innocent:

**Comment by dpobel** [comment]
+1 just make sure to log a JIRA issue for the untranslatable messages

**Comment by lolautruche** [comment]
Here it is: https://jira.ez.no/browse/EZP-24793
